examples/OECC/scripts/start.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Start the OpenEdge Command Center environment.
+#
+
+if ! /psc/dlc/bin/proutil /psc/wrk/sports2020 -C probe liveness
```

Comment:
This is one of those scripts where things seem to be hardcoded to `/psc/*` but may be better as just $DLC and $WRK with checks to ensure those paths are set and exist on disk. If we can do some quick checks of the variables and locations, then the script can exit with an error if they are somehow not set and the user will know that their paths are incorrect.

Reply:
The expectation is that the demo scripts perform the installation of OpenEdge to the known location.
Updated scripts to use environment variables.